### DISCUSSION TITLE
Is it possible to customize the search panel button icons?

### DISCUSSION THREAD

**Discussion by FMozingoCree** [icsharpcode/AvalonEdit#306] [Q&A] [answered]
I need to customize the dropdown search panel to match my application but I can seem to figure out how. I am using C# WPF .net Framework 4.7.

**Comment by maxijabase** [comment, marked as answer]
Take a look at the following Google search:
https://www.google.com/search?q=avalonedit+search+panel+style

You may be interested in http://arbitrarybytes.blogspot.com/2014/09/avalonedit-search-panel-customization.html

**Reply by FMozingoCree** [reply]
The issue I had was that the arrow buttons had a formatting that made them unreadable. It turned out to be a global button style I had implemented and forgotten about. I greatly appreciate the response and information.
